apps/server/src/modules/teams/service/team.service.ts

```diff
@@ -22,6 +26,8 @@ export class TeamService {
 
 		await this.teamsRepo.save(teams);
 
+		this.logger.log({ action: 'Deleted users data from Teams for user ', userId });
```

Comment:
Please log some more metadata (as for the _classes_ cases where I've left some example).

Reply:
OK I can only now add info about number of updated teams without userId. More deteiled information about updated documents will be added in ticket https://ticketsystem.dbildungscloud.de/browse/BC-6114